client/FxAccountClient.js

```diff
@@ -1239,6 +1244,28 @@ define([
   };
 
   /**
+   * Consume a signinCode.
+   *
+   * @method consumeSigninCode
+   * @param {String} signinCode The signinCode entered by the user
+   * @param {String} flowId Identifier for the current event flow
+   * @param {Number} flowBeginTime Timestamp for the flow.begin event
```

Comment:
I opted to pass the metricsContext properties inidividually here to more clearly stress that they are not optional on this endpoint. For every other endpoint they can be ditched, but that is not the case here.